test/blackbox-tests/test-cases/enabled_if/eif-dune-describe-crash.t

```diff
@@ -0,0 +1,46 @@
+  $ cat > dune <<EOF
+  > (executable
+  >  (name test)
+  >  (enabled_if
+  >   (= %{system} macosx)))
+  > (rule
+  >  (enabled_if
+  >   (= %{system} macosx))
+  >  (alias runtest)
+  >  (action
+  >  (run ./test.exe)))
+  > EOF
+
+  $ cat > test.ml <<EOF
+  > let () = print_string "Hello world"
+  > EOF
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.13)
+  > (package
+  >  (name pack)
+  >  (allow_empty))
+  > EOF
+
+  $ dune build
+  $ dune runtest
+  $ dune describe
+  Internal error, please report upstream including the contents of _build/log.
+  Description:
+    ("modules_and_obj_dir: failed lookup",
+    { keys = []; for_ = Exe { first_exe = "test" } })
+  Raised at Stdune__Code_error.raise in file
```

Comment:
Please remove the stacktrace from the test output.

Reply:
Done.